Command/GenerateCommand.php

```diff
@@ -20,7 +20,7 @@ class GenerateCommand extends AbstractCommand
     private $availableMigrationFormats = array('yml', 'php', 'sql', 'json');
     private $availableModes = array('create', 'update', 'delete');
     private $availableTypes = array('content', 'content_type', 'content_type_group', 'language', 'object_state', 'object_state_group', 'role', 'section', 'generic', 'db', 'php', '...');
-    private $thisBundle = 'EzMigrationBundle';
+    private $thisBundle = 'eZMigrationBundle';
```

Comment:
Does this cause trouble with autoloading? Or something else?

Reply:
Yes, I had
```
!!                                                                                 
!!    Case mismatch between loaded and declared class names: "Kaliop\eZMigrationBundle\eZMigrationBundle" vs "Kaliop\eZMigrationBundle\EzMigrationBundle".    
!!               
```
composer added a broken namespace to bundles.php, but it looks ok now